docs/doc.md

```diff
@@ -0,0 +1,8 @@
+# My Favorite Markdown File
+
+Title: {{.Title}}
+Type: {{.Type}}
+
+## Properties
+
+Description: {{.Properties.Core.Properties.Title.Description}}
```

Comment:
@alarsyo cannot see the autogenerated file text in place of this file text.

Reply:
Whoops I forgot to push it 😁